apps/sms/test/marionette/conversation_test.js

```diff
@@ -491,4 +503,101 @@ marionette('Conversation Panel Tests', function() {
+      // several recipients
+      newMessageView.typeMessage('message2');
+      newMessageView.addNewRecipient('+234');
+
+      newMessageView.showSubject();
+      newMessageView.typeSubject('subject1');
+      conversationView = newMessageView.send();
+
+      newMessageView = conversationView.addRecipients();
+      assert.equal(newMessageView.recipients.length, 2,
+        'There should be 2 recipients');
+      assert.include(newMessageView.recipients, '+123',
+        '+123 should be in the recipients');
+      assert.include(newMessageView.recipients, '+234',
+        '+234 should be in the recipients');
+      assert.equal(newMessageView.messageText, '', 'MMS text should be empty');
+    });
+
+    test('Sms being written: message and recipients should be kept',
+    function() {
+      conversationView.typeMessage('message2');
+      var newMessageView = conversationView.addRecipients();
+      assert.equal(newMessageView.recipients, '+123',
+        'Recipient should be +123');
+      assert.equal(newMessageView.messageText, 'message2',
+        'message text should be kept');
+    });
+
+    test('Mms being written: all info must be kept', function() {
+      conversationView.typeMessage('message2');
+      // add a subject
+      conversationView.showSubject();
+      conversationView.typeSubject('subject1');
+      var newMessageView = conversationView.addRecipients();
+
+      // assert that we kept all the infos
+      assert.equal(newMessageView.recipients, '+123',
+        'Recipient should be +123');
+      assert.equal(newMessageView.messageText, 'message2',
+        'Message text should be kept');
+      assert.equal(newMessageView.subject, 'subject1',
+        'Subject should be kept');
+
+      // let's add a recipients and send the message.
+      // we will have a thread with several recipients
+      newMessageView.addNewRecipient('+234');
+      conversationView = newMessageView.send();
+      // write another MMS
+      conversationView.typeMessage('message3');
+      conversationView.showSubject();
+      conversationView.typeSubject('subject2');
+      // add a blob
+      conversationView.addAttachment().choose('Messages Activity Caller')
+        .pickImage();
+      messagesApp.switchTo();
+
+      // redirect to new message to add recipients
+      newMessageView = conversationView.addRecipients();
+
+      // assert the content
+      assert.equal(newMessageView.recipients.length, 2,
+        'There should be 2 recipients');
+      assert.include(newMessageView.recipients, '+123',
+        '+123 should be in the recipients');
+      assert.include(newMessageView.recipients, '+234',
+        '+234 should be in the recipients');
+      assert.equal(newMessageView.messageText, 'message3',
+        'Message text should be kept');
+      assert.equal(newMessageView.subject, 'subject2',
+        'Subject should be kept');
+      assert.ok(newMessageView.attachments, 'Attachment should be kept');
```

Comment:
Ok let's screenshot!

First one, so far so good
![screen_attachment_before](https://cloud.githubusercontent.com/assets/1576598/11607203/baf9ec6c-9b3d-11e5-98ae-2c323cbc25a3.png)

Second one, nooooo !!
![screen_attachment_after](https://cloud.githubusercontent.com/assets/1576598/11607202/b7588c08-9b3d-11e5-90e5-388911b1ab13.png)

Indeed, I focus the keyboard after redirecting to the new message view, and apparently, nothing is focused just after adding an attachment.

So I thought of 3 different ways to circumvent that
- Make the keyboard disappear. Is there a standard way to do that other than switching to keyboard iframe, long-tapping on the right spot and switching back to sms iframe?
- tap the empty `#messages-container`. This fails with the very unhelpful `Crash detected but error running stackwalk` message
- make the window bigger (at least vertically). But maybe we test a particular size on purpose? And is it even possible to specify the startup size?

Do you have another idea?